src/test/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/changelist/commonmodels/CompanyIdentificationTest.java

```diff
@@ -0,0 +1,87 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.commonmodels;
```

Comment:
Maybe this Test class should be in another PR as not related to changes.

Reply:
This test is now here https://github.com/companieshouse/overseas-entities-api/pull/256